posts/MichelaRocchettiBlogPost/Michela_JSoC.qmd

```diff
@@ -0,0 +1,111 @@
+provide useful utilities in this package to manipulate IPUMS data sources,
+ and compose this package with the broader Julia ecosystem to enable novel research in health, economics, and more. In a long term view, 
+ IPUMS.jl can be used as a viable alternative to IPUMS.
+
+ The informations and the data collected by IPUMS are useful for comparative research, as well as analysis of indiviudals in their living context and 
+ those data can be used for crating a more comprehensive dataset to be at disposal, facilitating research into social determinant of health data for different types of 
+ diseases, social communities and geographical areas.
+
+
+# Tasks and Goals
+
+1. Expand some of the functionality drafted in ipumsr  IPUMS NHGIS
+    - Create a link between OpenAPI documentation and the functions internally used in IPUMS.jl:
+    updating already present function, determine if updating is needed and test them
+    - Develop functionality similar to the get_metadata_nghis function present in ipumsr
+
+2. Update IPUMS documentation
+    - Set-up and deploy of DocumenterVitepress.jl  
+    - Write a blog post on how IPUMS.jl can compose within the ecosystem.
+
+
+# Workflow 
+The first task was to migrate documents from Documenter to DocumenterVitepress.The aim of this issue is to assist with the significant refactor underway across JuliaHealth, 
+aimed at enhancing the discoverability and cohesion of the JuliaHealth ecosystem, particularly in relation to documentation.This issue is intended to offer guidance on migrating
+the IPUMS.jl documentation site from Documenter.jl to DocumenterVitepress.jl, in order to create a more engaging entry point for new Julia users interested in health research within the Julia community.
+To complete this task, what was done was to add a dependency of DocumenterVitepress into the docs directory of the IPUMS.jl repository. 
+When this was completed, the file Documenter.jl make.jl were migrated into a DocumenterVitepress.jl make.jl file. Working on the make.jl file pages, paragraphs and subparagraphs were added, in order to have
+a structure for the webpage explaining the IPUMS.jl package. In this prespective, some pages were added:
+  1. Home: to explain the main purpose of this work and to give general informations.
+  2. Workflows: to explain the work process behind.
+  3. How to: to give general information on how to make things work.
+  4. Tutorials: some tutorials on what is possible to do with IPUMS.jl and how to make it work 
+  5. Examples: some activity examples.
+  6. Mission: to better explain why IPUMS.jl package is useful for the community and its final objective and possible implications
+  7. References: references used to write the pages down.
+
+This first task requires some time, in particular to set up github  and cloning the repository locally in a correct manner.
```

Comment:
Oh could you talk a bit about some more of the issues you encountered here were?